scripts/build/build/targets.py

```diff
@@ -68,7 +68,7 @@ def BuildHostFakeTarget():
     target.AppendModifier("asan", use_asan=True).ExceptIfRe("-tsan")
     target.AppendModifier("tsan", use_tsan=True).ExceptIfRe("-asan")
     target.AppendModifier("ubsan", use_ubsan=True)
-    target.AppendModifier("libfuzzer", use_tsan=True).OnlyIfRe("-clang")
+    target.AppendModifier("libfuzzer", use_libfuzzer=True).OnlyIfRe("-clang")
```

Comment:
This seems unrelated to the Python controller.

Reply:
yes its unrelated, mistakenly pulled latest changes from master  via clicking update branch button on github..